### ISSUE TITLE
VSCode break the running debug session on launch.

### ISSUE THREAD

**Issue by IvanGrab** [rokucommunity/roku-debug#69]
Roky OS: v11.0.0
VSCode: v1.65.0 / Commit: b5205cc8eb4fbaa726835538cd82372cc0222d43
BrightScript Language extension: v2.28.1

But its worked with side-load
Logs:
```
Thread selected:  0*   pkg:/source/SGDEX.brs(4)                m.args = args
Protocol Version 3.0.0 has not been tested and my not work as intended.
Please open any issues you have with this version to https://github.com/rokucommunity/roku-debug/issues
```

**Comment by chrisdp** [comment]
@IvanGrab support for v3.0.0 of the protocol is under active development and we hope to release it in a few days after some internal validation. 

See pr: https://github.com/rokucommunity/roku-debug/pull/45

**Comment by TwitchBronBron** [comment]
@IvanGrab in the mean time, you can use the telnet debugger by setting `"enableDebugProtocol": false` in your launch.json

**Comment by IvanGrab** [comment]
> @IvanGrab in the mean time, you can use the telnet debugger by setting `"enableDebugProtocol": false` in your launch.json

Thanks a lot, also want to mention thats in pr: https://github.com/rokucommunity/roku-debug/pull/45 it fixed for me.

So maybe close the issue?

**Comment by chrisdp** [comment]
@IvanGrab we will close it when we release it to everyone.

**Comment by TwitchBronBron** [comment]
This was released in roku-debug@0.10.0, which was included in vscode-brightscript-language@2.29.0.
